DiscordBot/Modules/AdminModule.cs

```diff
@@ -17,12 +17,24 @@ public AdminModule(IInitializeGuilds guildRepo)
         }
 
         [Command("Init")]
-        public Task InitializeGuildAsync()
+        public async Task InitializeGuildAsync()
         {
             var guild = Context.Guild;
-            _guildRepo.AddNewGuild(guild);
-            //_guildRepo.SaveGuildUsers(guild);
-            return Task.FromResult(true);
+
+            var successfullyAddedGuild = await _guildRepo.AddNewGuild(guild);
+
+            if (successfullyAddedGuild)
+            {
+                await _guildRepo.SaveGuildUsers(guild);
+            }
+        }
+
+        [Command("Update users")]
```

Comment:
Is this command going to need to be run each time to update the list of users? Or will the bot be able to update users automatically?

Reply:
Hmm i could probably make the code more clear.  This command calls `_guildRepo.UpdateGuildUsers(guild)` which is also called when you initialize a guild.  To answer your question: the list of users should automatically remain updated in real time (user joining, user leaving, etc - we still need to build all that functionality).  This specific command would ideally only be used in reconciliation situations, aka if the bot was down for some reason and we missed some users joining or leaving.